docs/reference/faq.md

```diff
@@ -219,8 +219,8 @@ This is the canonical list of keyboard shortcuts:
 		</tr>
 		<tr>
 			<td>Remove a link.</td>
-			<td><kbd>Shift</kbd>+<kbd>Alt</kbd>+<kbd>S</kbd></td>
-			<td><kbd>⌃</kbd><kbd>⌥</kbd><kbd>S</kbd></td>
+			<td><kbd>Shift</kbd>+<kbd>Alt</kbd>+<kbd>K</kbd></td>
```

Comment:
You forgot something here.

Reply:
Good catch. Did 7c5f43cdf129679a34f551a1eea1667134c370fa fix it? I'm unsure about the proper keys for Mac - I've just copied what other shortcuts had for the Ctrl+Shift combination.